debug-tools/debug-tools.nix

```diff
@@ -0,0 +1,39 @@
+{ lib,
+  stdenv,
+  ninja,
+  pkg-config,
+  cmake,
+  crypto3,
+  boost,
+  gdb,
+  cmake_modules,
+  enableDebugging,
+  enableDebug ? false,
+  runTests ? false,
```

Comment:
we don't need this flag, as it does not have tests

Reply:
removed three flags. `cmake_modules` must be preserved